terraso_backend/apps/core/models/groups.py

```diff
@@ -45,6 +47,11 @@ class Group(SlugModel):
 
     field_to_slug = "name"
 
+    class Meta:
+        rules_permissions = {
+            "change": perm_rules.is_group_manager,
```

Comment:
Question: Is it possible for a a rule to exist on something that is not a `Model`? For example `Download all stored images` that will be a permission on the `storage` app, is something like that supported by this?